app/src/main/java/com/example/marvelapp/fragment/CharacterDetailFragment.kt

```diff
@@ -0,0 +1,80 @@
+package com.example.marvelapp.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import com.bumptech.glide.Glide
+import com.example.marvelapp.R
+import com.example.marvelapp.databinding.FragmentCharacterDetailBinding
+import com.example.marvelapp.entity.Character
+import com.example.marvelapp.mvvm.viewmodel.CharacterDetailFragmentViewModel
+import org.koin.android.ext.android.inject
+import org.koin.core.component.KoinComponent
+
+class CharacterDetailFragment : DialogFragment(), KoinComponent {
+
+    private lateinit var binding: FragmentCharacterDetailBinding
+    private val viewModel: CharacterDetailFragmentViewModel by inject()
+    private lateinit var characterId: String
+
+    override fun onCreate(savedInstanceState: Bundle?) {
```

Comment:
please order correctly the lines in the onCreate

Reply:
done